openff/interchange/components/interchange.py

```diff
@@ -292,6 +300,7 @@ def to_top(self, file_path: Union[Path, str], writer="internal"):
             to_top(self, file_path)
 
     def to_lammps(self, file_path: Union[Path, str], writer="internal"):
+        """Write this Interchange to a LAMMPS data file."""
```

Comment:
Any reason this method says "Write" when others say "Export"? This is the nittiest of nitpicks...

Reply:
Not a nitpick - I think the consistency would be valuable

In general, my brain associates "write" with file and "export" with in-memory objects, but that distinction is not robust.

Reply:
```suggestion
        """Export this Interchange to a LAMMPS data file."""
```